WB_sRGB_Python/classes/WBsRGB.py

```diff
@@ -163,11 +148,10 @@ def kernelP(I):
         Ref: Hong, et al., "A study of digital camera colorimetric
           characterization based on polynomial modeling." Color Research &
           Application, 2001. """
-  return (np.transpose(
-    (I[:, 0], I[:, 1], I[:, 2], I[:, 0] * I[:, 1], I[:, 0] * I[:, 2],
-     I[:, 1] * I[:, 2], I[:, 0] * I[:, 0], I[:, 1] * I[:, 1],
-     I[:, 2] * I[:, 2], I[:, 0] * I[:, 1] * I[:, 2],
-     np.repeat(1, np.shape(I)[0]))))
+  rgb = I
```

Comment:
We may need to change kernelP(I) to kernelP(rgb)

Reply:
Maybe we can apply the change in other commits so that the PR won't introduced backward incompatible changes. The users might call the public function `kernelP` with keyword argument `KernelP(I=some_rgb_image)`.